### PR TITLE
Added Convert Minutes to Hours in JavaScript

### LINKED ISSUE #3511: Write a JavaScript program to convert minutes to hours
### Description

Write a JavaScript program to convert minutes to hours

1 hour = 60 minutes

```
Input  : 300
Output : 5
```
<details>
<summary>How to contribute</summary>

- Save the solution in `program/ConvertMinutesToHours.js` file
- Add `ConvertMinutesToHours.js` file in `convert-minutes-to-hours` folder

</details>

### PR DESCRIPTION
**Pull request by sidharthbh8**
### Why:

Closes codinasion/codinasion#2580

### What's being changed:

Added Convert Minutes to Hours in JavaScript

### CODE REVIEW

**Review by sidharthbh8 — COMMENTED** [review]
reviewed

**Comment by sidharthbh8** [comment]
@joao-vitor-souza  can you tell me why its showing merging blocked

File: program/ConvertMinutesToHours.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    let hours = minute/60
+    return hours;
+}
+
+// Test Console
+console.log(`${ConvertMinutesToHours(300)}Hours`);
+
+console.log(`${ConvertMinutesToHours(120)}Hours`);
+
+console.log(`${ConvertMinutesToHours(60)}Hours`);
```
Comment (victoriacheng15) [inline program/ConvertMinutesToHours.js:13]: The output is `1hours`.  You can make it output `hour` or `hours` based on the result you get.

**Review by victoriacheng15 — DISMISSED** [review]

**Comment by victoriacheng15** [comment]
> @joao-vitor-souza can you tell me why its showing merging blocked

This PR was not reviewed by one of us yet.

**Review by harshraj8843 — APPROVED** [review]
Thanks for contributing 👍🏻

**Pull request merged**